### DISCUSSION TITLE
In example local testing colors dosen't work

### DISCUSSION THREAD

**Discussion by SoulNaturalist** [Ashutosh00710/github-readme-activity-graph#202] [Q&A] [answered]
![image](https://github.com/Ashutosh00710/github-readme-activity-graph/assets/58743402/37c628f8-0285-422f-a14f-1c6637f946cf)
![image](https://github.com/Ashutosh00710/github-readme-activity-graph/assets/58743402/2edd1ed5-0c53-4c60-918f-9745e45cb431)
When I run the application locally it writes `Something unexpected happened 💥` what am I missing
even when I change the example, the exact same error comes out

**Comment by Ashutosh00710** [comment, marked as answer]
Have you generated the token and placed it in `.env` file?

**Reply by SoulNaturalist** [reply]
I forgot to replace it with my github token, thank you
